src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -289,6 +286,15 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
       this.resetSelection = false;
     }
 
+    const globalScratchData: CytoscapeGlobalScratchData = {
+      edgeLabelMode: this.props.edgeLabelMode,
+      graphType: this.props.graphType,
+      showNodeLabels: this.props.showNodeLabels,
+      versioned: this.props.versioned
+    };
+    console.debug('cy global scratch: ' + JSON.stringify(globalScratchData));
```

Comment:
Is the `console.debug` left here on purpose?

Reply:
I actually had this to debug this problem we were having in the Enum stuff :) I will remove

Reply:
damn Enum stuff!